dashboard/app/views/levels/_form.html.haml

```diff
@@ -29,9 +29,9 @@
     = render partial: 'levels/editors/karel', locals: {f: f} if @level.is_a? Karel
     = render partial: 'levels/editors/artist', locals: {f: f} if @level.is_a? Artist
     = render partial: 'levels/editors/cs_in_a', locals: {f: f} if (@level.is_a?(Calc) || @level.is_a?(Eval))
-    = render partial: 'levels/editors/applab', locals: {f: f} if @level.is_a? Applab
-    = render partial: 'levels/editors/gamelab', locals: {f: f} if @level.is_a? Gamelab
-    = render partial: 'levels/editors/gamelab_jr', locals: {f: f} if @level.is_a? GamelabJr
+    = render partial: 'levels/editors/gamelab', locals: {f: f} if @level.is_a?(Gamelab) && !@level.is_a?(GamelabJr)
```

Comment:
looks like we lost applab on this line. the PR description didn't mention this so maybe this was a mistake?

Reply:
Oh no, that's a mistake. Thank you for catching!